physics/ephemeris.hpp

```diff
@@ -32,6 +32,7 @@ class Ephemeris {
   // The equation describing the motion of the |bodies_|.
   using NewtonianMotionEquation =
       SpecialSecondOrderDifferentialEquation<Position<Frame>>;
+  using Trajectories = std::vector<not_null<Trajectory<Frame>*>>;
```

Comment:
I am not too happy with this change, as it seems rather random to put this here since Ephemeris doesn't really deal with Trajectory objects.  When I saw a reference in Plugin I had to look to see if this was a continuous or a discrete trajectory.  I suggest moving the typedef to plugin.h.